packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -3253,9 +3263,11 @@ describe('TableView', function () {
       let table = tree.getByRole('grid');
       let columnheaders = within(table).getAllByRole('columnheader');
       expect(columnheaders).toHaveLength(3);
-      expect(columnheaders[0]).toHaveAttribute('aria-sort', 'descending');
-      expect(columnheaders[1]).toHaveAttribute('aria-sort', 'none');
-      expect(columnheaders[2]).not.toHaveAttribute('aria-sort');
+      expect(columnheaders[0]).toHaveAttribute('aria-describedby');
+      expect(document.getElementById(columnheaders[0].getAttribute('aria-describedby'))).toHaveTextContent('sortable column, descending');
```

Comment:
`aria-describedby` should contain a space-delimited list of IDREFs for elements in the DOM that describe or provide help text to the element.